### PR TITLE
fixing e501 for tests/commands

### PR DESCRIPTION
**Pull request by JohananOppongAmoateng**
This pr is a continuation of fixes for the  e501 issues in #2383

## PR Checklist:

- [ ] All new features have been tested
- [ ] All new features have been documented
- [x] I have read the **CONTRIBUTING.md** file
- [x] I will abide by the code of conduct

### CODE REVIEW

File: tests/commands/base/test_update_cookiecutter_cache.py [hunk 1]
```diff
@@ -131,30 +132,24 @@ def test_new_repo_invalid_template_url(base_command, mock_git):
     ("stderr_string", "error_message"),
     [
         pytest.param(
-            "\n    stderr: '\nfatal: could not clone repository 'https://example.com' \n'",
+            "\n    stderr: '\nfatal: could not clone repository"
+            " 'https://example.com' \n'",
```
Comment (freakboy3742) [inline tests/commands/base/test_update_cookiecutter_cache.py]: As noted in the past PR - when we've got a string as one argument of many, it's preferable to use bracket notation to highlight that this is one string argument, not two:
```suggestion
            (
                "\n    stderr: '\nfatal: could not clone repository"
                " 'https://example.com' \n'"
            ),
```

File: tests/commands/base/test_update_cookiecutter_cache.py [hunk 2]
```diff
@@ -131,30 +132,24 @@ def test_new_repo_invalid_template_url(base_command, mock_git):
...
-            ),
-            (
-                "Could not read from remote repository.\n\nPlease make sure "
-                "you have the correct access rights\nand the repository exists."
-            ),
+            "\n    stderr: '\nfatal: Could not read from remote repository.\n\nPlease"
+            " make sure you have the correct access rights\nand the repository"
+            " exists. \n'",
+            "Could not read from remote repository.\n\nPlease make sure "
+            "you have the correct access rights\nand the repository exists.",
```
Comment (freakboy3742) [inline tests/commands/base/test_update_cookiecutter_cache.py]: As above - these two should be bracket-wrapped

File: tests/commands/base/test_update_cookiecutter_cache.py [hunk 3]
```diff
@@ -131,30 +132,24 @@ def test_new_repo_invalid_template_url(base_command, mock_git):
...
-                "OpenSSL/3.2.2: error:0A000438:SSL routines::tlsv1 alert internal error'"
-            ),
-            (
-                "Unable to access 'https://example.com/': OpenSSL/3.2.2: "
-                "error:0A000438:SSL routines::tlsv1 alert internal error."
-            ),
+            "\n    stderr: '\nfatal: unable to access 'https://example.com/': "
+            "OpenSSL/3.2.2: error:0A000438:SSL routines::tlsv1 alert internal error'",
+            "Unable to access 'https://example.com/': OpenSSL/3.2.2: "
+            "error:0A000438:SSL routines::tlsv1 alert internal error.",
```
Comment (freakboy3742) [inline tests/commands/base/test_update_cookiecutter_cache.py]: As above - these two should be bracket-wrapped

File: tests/commands/create/test_create_app.py [hunk 4]
```diff
@@ -49,7 +49,8 @@ def test_create_existing_app_overwrite(tracking_create_command, tmp_path):
 
     # Input was required by the user
     assert tracking_create_command.console.prompts == [
-        f"The directory {bundle_path.relative_to(base_path)} already exists; overwrite [y/N]? "
+        f"The directory {bundle_path.relative_to(base_path)} already exists; overwrite"
+        " [y/N]? "
```
Comment (freakboy3742) [inline tests/commands/create/test_create_app.py]: Examples like this one are a bit of a judgement call as to whether extra parentheses are needed, as it's a list with one item; I'd be inclined to they should, to reinforce that it's a single prompt in a list.

(Similarly for the 2 other equivalent changes in this file)

File: tests/commands/create/test_install_stub_binary.py [hunk 5]
```diff
@@ -265,7 +265,7 @@ def test_install_custom_stub_binary_url(
     # Confirm the right file was unpacked
     create_command.tools.shutil.unpack_archive.assert_called_with(
         filename=tmp_path
-        / "data/stub/986428ef9d5a1852fc15d4367f19aa328ad530686056e9d83cdde03407c0bceb/My-Stub.zip",
+        / "data/stub/986428ef9d5a1852fc15d4367f19aa328ad530686056e9d83cdde03407c0bceb/My-Stub.zip",  # noqa: E501
```
Comment (freakboy3742) [inline tests/commands/create/test_install_stub_binary.py]: Preference here would be (a) to use brackets to clarify the scope of the `filename` argument, and (b) to break the string in more places to avoid the need for the `noqa`.  Github won't let me code suggest here, but something like:
```
        filename=(
            tmp_path
            / "data/stub"
            / "986428ef9d5a1852fc15d4367f19aa328ad530686056e9d83cdde03407c0bceb"
            / "My-Stub.zip"
        ),
```

File: tests/commands/new/test_build_gui_context.py [hunk 6]
```diff
@@ -38,7 +38,8 @@ def test_toga_bootstrap(new_command):
     )
 
     assert context == {
-        "app_source": '''\
+        "app_source": (
+            '''\
```
Comment (freakboy3742) [inline tests/commands/new/test_build_gui_context.py]: Are these brackets being added automatically by ruff? I can't spot an obvious reason why this file has so many changes - the brackets aren't allowing extra line breaks, and don't add any particular clarity. If they *are* being automatically added, or there's a line length issue that I've missed, then they're fine to include - but if they've been added manually, they're slightly overkill.

File: tests/commands/upgrade/test_call.py [hunk 7]
```diff
@@ -36,8 +36,8 @@ def test_list_tools(upgrade_command, mock_tool_registry, capsys):
         tools=upgrade_command.tools, install=False
     )
 
-    assert capsys.readouterr().out == (
-        "\n"
+    assert (
+        capsys.readouterr().out == "\n"
```
Comment (freakboy3742) [inline tests/commands/upgrade/test_call.py]: This is an odd change - the brackets are the exact opposite of what I would have expected. The reasoning is the same as for other bracket usage - it's not intuitive to tell which statements are the start of a string, and which the start of a new argument. The previous formatting made this clear - `copses.readouterr().out = (...some multiline string...)`; the new formatting is ambiguous.

File: tests/config/test_merge_pep621_config.py [hunk 8]
```diff
@@ -320,7 +322,8 @@ def test_optional_non_test_dependencies():
 
 
 def test_test_dependencies_without_requires():
-    "If the global config doesn't specify test requirements, test dependencies are used as is"
+    """If the global config doesn't specify test requirements, test " "dependencies are
```
Comment (freakboy3742) [inline tests/config/test_merge_pep621_config.py]: A stray pair of quotes, probably from a line break that existed at one point:
```suggestion
    """If the global config doesn't specify test requirements, test dependencies are
```

File: tests/console/Console/test_textwrap.py [hunk 9]
```diff
@@ -9,42 +9,50 @@
...
-            "There is nothing wrong with your television set.\n"
-            "Do not attempt to adjust the picture.",
+            (
+                "There is nothing wrong with your television set.\n"
+                "Do not attempt to adjust the picture."
+            ),
+            (
+                "There is nothing wrong with your television set.\n"
+                "Do not attempt to adjust the picture."
+            ),
```
Comment (freakboy3742) [inline tests/console/Console/test_textwrap.py:19]: +1 to this change - this is the preferred format for multi-line strings.

File: tests/console/Console/test_wait_bar.py [hunk 10]
```diff
@@ -18,8 +18,8 @@ def test_wait_bar_done_message_non_interactive(non_interactive_console, capsys):
     with non_interactive_console.wait_bar("Wait message...", done_message="finished"):
         pass
 
-    assert capsys.readouterr().out == (
-        "Wait message... started\nWait message... finished\n"
+    assert (
+        capsys.readouterr().out == "Wait message... started\nWait message... finished\n"
     )
```
Comment (freakboy3742) [inline tests/console/Console/test_wait_bar.py:23]: This is a bit of a value judgement - if an automated tool is doing this, then fine; but otherwise the older format would be preferred.

File: tests/console/test_NotDeadYet.py [hunk 11]
```diff
@@ -21,8 +21,9 @@ def test_update(capsys, monkeypatch, dummy_console):
         keep_alive.update()
         keep_alive.update()
 
-    assert capsys.readouterr().out == (
-        "... still waiting\n... still waiting\n... still waiting\n"
+    assert (
+        capsys.readouterr().out
+        == "... still waiting\n... still waiting\n... still waiting\n"
```
Comment (freakboy3742) [inline tests/console/test_NotDeadYet.py]: Again - the older format would be preferred unless this is a tool automated change.

**Comment by freakboy3742** [comment]
As a procedural thing - can I ask that you *don't* mark conversations as resolved (or, rather, that you only mark a conversation as resolved if your the person who *started* the conversation). When I (or another reviewer) provides a comment, that usually means we intend to review later work and ensure that the issues that were found have been resolved - *that* is when the conversation is resolved. Otherwise, the reviewer loses the context that they asked for a change, and it's not necessarily clear if the way you've responded to the comment was what was the reviewer intended.

Otherwise, it looks like we're 1 pre-commit violation statement away from this being reviewable. If you make that fix, I'll take another pass at it.

**Comment by JohananOppongAmoateng** [comment]
> As a procedural thing - can I ask that you _don't_ mark conversations as resolved (or, rather, that you only mark a conversation as resolved if your the person who _started_ the conversation). When I (or another reviewer) provides a comment, that usually means we intend to review later work and ensure that the issues that were found have been resolved - _that_ is when the conversation is resolved. Otherwise, the reviewer loses the context that they asked for a change, and it's not necessarily clear if the way you've responded to the comment was what was the reviewer intended.
> 
> Otherwise, it looks like we're 1 pre-commit violation statement away from this being reviewable. If you make that fix, I'll take another pass at it.

okay thanks for letting me know

**Comment by JohananOppongAmoateng** [comment]
@freakboy3742  @kattni sorry this pr has really been slow i have to work on some other stuff since  last week hoping i can free up sometime this week and get this over the finish line

File: tests/commands/base/test_update_cookiecutter_cache.py [hunk 12]
```diff
@@ -131,34 +132,36 @@ def test_new_repo_invalid_template_url(base_command, mock_git):
...
+                "OpenSSL/3.2.2: error:0A000438:SSL routines::tlsv1 alert internal "
+                "error'"
             ),
+            "Unable to access 'https://example.com/': OpenSSL/3.2.2: "
+            "error:0A000438:SSL routines::tlsv1 alert internal error.",
             id="no-tailing-whitespace-no-caps-no-period",
         ),
         pytest.param(
-            "\n stderr: 'Mysterious git clone edge case with no fatal error.",
+            ("\n stderr: 'Mysterious git clone edge case with no fatal error."),
```
Comment (freakboy3742) [inline tests/commands/base/test_update_cookiecutter_cache.py]: The brackets are redundant here.
```suggestion
            "\n stderr: 'Mysterious git clone edge case with no fatal error.",
```

File: tests/commands/base/test_update_cookiecutter_cache.py [hunk 13]
```diff
@@ -131,34 +132,36 @@ def test_new_repo_invalid_template_url(base_command, mock_git):
...
-                "OpenSSL/3.2.2: error:0A000438:SSL routines::tlsv1 alert internal error'"
-            ),
-            (
-                "Unable to access 'https://example.com/': OpenSSL/3.2.2: "
-                "error:0A000438:SSL routines::tlsv1 alert internal error."
+                "OpenSSL/3.2.2: error:0A000438:SSL routines::tlsv1 alert internal "
+                "error'"
             ),
+            "Unable to access 'https://example.com/': OpenSSL/3.2.2: "
+            "error:0A000438:SSL routines::tlsv1 alert internal error.",
```
Comment (freakboy3742) [inline tests/commands/base/test_update_cookiecutter_cache.py]:
```suggestion
            (
                "Unable to access 'https://example.com/': OpenSSL/3.2.2: "
                "error:0A000438:SSL routines::tlsv1 alert internal error."
            ),
```

File: tests/commands/base/test_update_cookiecutter_cache.py [hunk 14]
```diff
@@ -131,34 +132,36 @@ def test_new_repo_invalid_template_url(base_command, mock_git):
...
-            (
-                "Could not read from remote repository.\n\nPlease make sure "
-                "you have the correct access rights\nand the repository exists."
+                "\n    stderr: '\nfatal: Could not read from remote repository."
+                "\n\nPlease"
+                " make sure you have the correct access rights\nand the repository"
+                " exists. \n'"
             ),
+            "Could not read from remote repository.\n\nPlease make sure "
+            "you have the correct access rights\nand the repository exists.",
```
Comment (freakboy3742) [inline tests/commands/base/test_update_cookiecutter_cache.py]:
```suggestion
        (
            "Could not read from remote repository.\n\nPlease make sure "
            "you have the correct access rights\nand the repository exists."
        ),
```

File: tests/commands/create/test_create_app.py [hunk 15]
```diff
@@ -49,7 +49,11 @@ def test_create_existing_app_overwrite(tracking_create_command, tmp_path):
 
     # Input was required by the user
     assert tracking_create_command.console.prompts == [
-        f"The directory {bundle_path.relative_to(base_path)} already exists; overwrite [y/N]? "
+        (
+            f"The directory {bundle_path.relative_to(base_path)} already exists;"
+            " overwrite"
+            " [y/N]? "
+        ),
```
Comment (freakboy3742) [inline tests/commands/create/test_create_app.py:55]:
```suggestion
        (
            f"The directory {bundle_path.relative_to(base_path)} already exists;"
            " overwrite [y/N]? "
        ),
```

File: tests/commands/create/test_create_app.py [hunk 16]
```diff
@@ -85,7 +89,11 @@ def test_create_existing_app_no_overwrite(tracking_create_command, tmp_path):
 
     # Input was required by the user
     assert tracking_create_command.console.prompts == [
-        f"The directory {bundle_path.relative_to(base_path)} already exists; overwrite [y/N]? "
+        (
+            f"The directory {bundle_path.relative_to(base_path)} already exists;"
+            " overwrite"
+            " [y/N]? "
```
Comment (freakboy3742) [inline tests/commands/create/test_create_app.py]:
```suggestion
            " overwrite [y/N]? "
```

File: tests/commands/create/test_create_app.py [hunk 17]
```diff
@@ -113,7 +121,11 @@ def test_create_existing_app_no_overwrite_default(tracking_create_command, tmp_p
...
```
Comment (freakboy3742) [inline tests/commands/create/test_create_app.py]:
```suggestion
            " overwrite [y/N]? "
```
